src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Interop/BrowserInteropHelper.cs

```diff
@@ -241,17 +240,7 @@ private static void EnsureScriptInteropAllowed()
                 _isScriptInteropDisabled.Value = SafeSecurityHelper.IsFeatureDisabled(SafeSecurityHelper.KeyToRead.ScriptInteropDisable);
             }
 
-            // Similar approach as with WebBrowser.cs.
-            if (_isScriptInteropDisabled.Value.Value)
-            {
-                // Feature is disabled - demand unrestricted WebBrowserPermission to hand out the script object.
-                MS.Internal.PresentationFramework.SecurityHelper.DemandWebBrowserPermission();
-            }
-            else
-            {
-                // Feature is enabled - demand Safe level to hand out the script object, granted in Partial Trust by default.
-                (new WebBrowserPermission(WebBrowserPermissionLevel.Safe)).Demand();
-            }
+            (new WebBrowserPermission(WebBrowserPermissionLevel.Safe)).Demand();
```

Comment:
We still have the demand here, is that intended?

Reply:
Yep, i've already removed this in a separate branch which will be the last PR I open (as the purpose of this PR was more targeted to internal permission types and `SecurityHelper`)